skimage/io/tests/test_pil.py

```diff
@@ -39,6 +39,13 @@ def setup_module(self):
     except ImportError:
         pass
 
+def test_png_round_trip():
+    f = NamedTemporaryFile(suffix='.png')
+    fname = f.name
```

Comment:
I was referring to the test failure on Appveyor, which was due to a permissions error on this file. 



—
Sent from Mailbox

On Sun, Aug 30, 2015 at 12:50 PM, Connelly Barnes
<notifications@github.com> wrote:

>> @@ -39,6 +39,13 @@ def setup_module(self):
>>      except ImportError:
>>          pass
>>  
>> +def test_png_round_trip():
>> +    f = NamedTemporaryFile(suffix='.png')
>> +    fname = f.name
> I don't think so. When f is garbage collected at the end of the function, the temporary file will be removed per the NamedTemporaryFile spec: "If delete is true (the default), the file is deleted as soon as it is closed."
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/scikit-image/scikit-image/pull/1687/files#r38274032